gcloud/datastore/batch.py

```diff
@@ -189,8 +189,8 @@ def delete(self, key):
         if not _dataset_ids_equal(self._dataset_id, key.dataset_id):
             raise ValueError("Key must be from same dataset as batch")
 
-        key_pb = key.to_protobuf()
-        helpers._add_keys_to_request(self.mutation.delete, [key_pb])
+        key_pb = helpers._prepare_key_for_request(key.to_protobuf())
```

Comment:
This change isn't about removing a `datastore.helpers` import:  it changes how the deleted key gets added to the mutation (effectively, in inlines `helpers._add_keys_to_request`).